Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM elixir:1.7.4-alpine
+FROM elixir:1.13.2-alpine
```

Comment:
this works fine with the current elixir version in `mix.exs` (1.10) but If we want to match I can change this to `FROM elixir:1.10.4-alpine`